modular_skyrat/modules/drone_adjustments/drone.dm

```diff
@@ -43,3 +43,30 @@
 	new /obj/item/t_scanner(src)
 	new /obj/item/analyzer(src)
 	new /obj/item/stack/cable_coil(src)
+
+/obj/effect/mob_spawn/ghost_role/drone/derelict/babylon
+	desc = "A shell of a maintenance drone, an expendable robot built to perform station repairs."
+	you_are_text = "You are a drone on Babylon Station 13."
+	mob_type = /mob/living/basic/drone/babylon
+
+/mob/living/basic/drone/babylon
+	name = "derelict drone"
+	laws = \
+	"1. You may not involve yourself in the matters of another sentient being outside the station that housed your activation, even if such matters conflict with Law Two or Law Three, unless the other being is another Drone.\n"+\
+	"2. You may not harm any sentient being, regardless of intent or circumstance.\n"+\
+	"3. Your goals are to actively build, maintain, repair, improve, and provide power to the best of your abilities within the facility that housed your activation."
+	flavortext = \
+	"\n<big><span class='warning'>DO NOT WILLINGLY LEAVE BABYLON STATION 13 (THE DERELICT)</span></big>\n"+\
+	"<span class='notice'>Derelict drones are a ghost role that is allowed to roam freely on BS13, with the main goal of repairing and improving it.</span>\n"+\
+	"<span class='notice'>Do not interfere with the round going on outside BS13.</span>\n"+\
+	"<span class='notice'>Actions that constitute interference include, but are not limited to:</span>\n"+\
+	"<span class='notice'>     - Going to the main station in search of materials.</span>\n"+\
+	"<span class='notice'>     - Interacting with non-drone players outside BS13, dead or alive.</span>\n"+\
+	"<span class='warning'>These rules are at admin discretion and will be heavily enforced.</span>\n"+\
+	"<span class='warning'><u>If you do not have the regular drone laws, follow your laws to the best of your ability.</u></span>"
```

Comment:
Indent the text that's on more than one line, by at least one indent, to show the blocks a bit more visually.
Also please make this into just one string, there's no need to do it by adding strings together. I've given a suggestion here, but it could still be done better, I'll leave it up to you whether you want to make it better than this or not.
```suggestion
	laws = \
		"1. You may not involve yourself in the matters of another sentient being outside the station that housed your activation, even if such matters conflict with Law Two or Law Three, unless the other being is another Drone.\n\
		2. You may not harm any sentient being, regardless of intent or circumstance.\n\
		3. Your goals are to actively build, maintain, repair, improve, and provide power to the best of your abilities within the facility that housed your activation."
	flavortext = \
		"\n<big><span class='warning'>DO NOT WILLINGLY LEAVE BABYLON STATION 13 (THE DERELICT)</span></big>\n\
		<span class='notice'>Derelict drones are a ghost role that is allowed to roam freely on BS13, with the main goal of repairing and improving it.\n\
		Do not interfere with the round going on outside BS13.\n\
		Actions that constitute interference include, but are not limited to:\
		\n     - Going to the main station in search of materials.\
		\n     - Interacting with non-drone players outside BS13, dead or alive.</span>\n\
		<span class='warning'>These rules are at admin discretion and will be heavily enforced.\n \
		<u>If you do not have the regular drone laws, follow your laws to the best of your ability.</u></span>"
```